testcontainers/compose.py

```diff
@@ -80,6 +83,16 @@ def stop(self):
             subprocess.call(["docker-compose", "-f", self.compose_file_name, "down", "-v"],
                             cwd=self.filepath)
 
+    def get_logs(self):
+        with blindspin.spinner():
```

Comment:
Thoughts on using `subprocess.run` rather than the low-level `Popen` as recommended [here](https://docs.python.org/3/library/subprocess.html#using-the-subprocess-module), i.e. something like this?

```python
result = subprocess.run(["docker-compose", "-f", self.compose_file_name, "logs"],
                        cwd=self.filepath, capture_output=True)
return result.stdout, result.stderr
```

Reply:
It makes sense. Fixed.